rest_tools/server/auth.py

```diff
@@ -46,7 +50,12 @@ def create_token(self, subject, expiration=None, type='temp', payload=None):
             'iat': now,
             'type': type,
         })
-        return jwt.encode(payload, self.secret, algorithm=self.algorithm).decode('utf-8')
+
+        token = jwt.encode(payload, self.secret, algorithm=self.algorithm)
+        try:
+            return token.decode("utf-8")
+        except (UnicodeDecodeError, AttributeError):
+            return token
```

Comment:
We can probably just drop `.decode('utf-8')` and be done with it, unless you think we should maintain compatibility with older versions of pyjwt.

Reply:
I'm fine either way. Not maintaining backwards compatibility could break scripts that are using rest-tools _and_ <2.0 pyjwt. I'm not sure if that's something we should be aware of

Reply:
I think most things are only using pyjwt through rest-tools.  There might be one or two special projects, but they shouldn't be special and indicates code should be upstreamed to rest-tools.

Reply:
So, I'm fine with upgrading and breaking old things, as long as you update `requirements.txt`